### PR TITLE
Split this XXX characters long line (which is greater than 130 authorized).

### PR DESCRIPTION
**Pull request by KevinBoulongne**
Some lines were too long.

### CODE REVIEW

**Comment by LunarX** [comment]
Not sure this requiring this check is always productive actually

**Review by tevincent — APPROVED** [review]

File: app/src/main/java/com/infomaniak/mail/data/api/FlatteningSubBodiesSerializer.kt [hunk 1]
```diff
@@ -22,7 +22,7 @@ import io.realm.kotlin.serializers.RealmListKSerializer
 import io.realm.kotlin.types.RealmList
 import kotlinx.serialization.json.*
 
-// Documentation: https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-json/kotlinx.serialization.json/-json-transforming-serializer/
+// Documentation on KotlinLang.org : https://chk.me/kYZiqN0
```
Comment (LunarX) [inline app/src/main/java/com/infomaniak/mail/data/api/FlatteningSubBodiesSerializer.kt:25]: Not sure about that solution as well

Comment (KevinBoulongne) [reply app/src/main/java/com/infomaniak/mail/data/api/FlatteningSubBodiesSerializer.kt:25]: What do you propose instead ?

Comment (LunarX) [reply app/src/main/java/com/infomaniak/mail/data/api/FlatteningSubBodiesSerializer.kt:25]: Yeah the comment wasn't very clear now that I read it again. I think we could keep that one on a single line as it doesn't obstruct reading

**Pull request merged**